src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -71,13 +71,11 @@ public sealed class TokenCache : ITokenCacheInternal
         private TokenCacheCallback _userConfiguredAfterAccess;
         private TokenCacheCallback _userConfiguredBeforeWrite;
 
-        internal IServiceBundle ServiceBundle { get; private set; }
+        internal IServiceBundle ServiceBundle { get; }
```

Comment:
Question, if the setter on a property isn't defined ("{ get; }" for example), wouldn't the property already be read only as the property cannot be set outside of the class?